hubconf.py

```diff
@@ -22,9 +22,10 @@
 from transformers import (
     AutoConfig,
     AutoModel,
+    AutoModelForCausalLM,
+    AutoModelForMaskedLM
     AutoModelForQuestionAnswering,
     AutoModelForSequenceClassification,
-    AutoModelWithLMHead,
```

Comment:
Since this class is deprecated, let's remove it from the hub (plus the new way it's created does not let it have a docstring since it subclasses another class to issue the deprecating warnings and the current file was failing).